.github/workflows/ci-push.yml

```diff
@@ -184,3 +184,36 @@ jobs:
         name: racketcs-${{ matrix.os }}-x64_git${{ github.sha }}
         path: ../racketcs-${{ matrix.os }}-x64_git${{ github.sha }}.tar.bz2
 
+# Tests
+  test-cgc:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-18.04, macos-latest]
```

Comment:
Why not windows?

Reply:
The build was not working when I wrote this. Will add it next.